pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -225,15 +231,20 @@ def test_constructor_positional(self):
 
     def test_constructor_keyword(self):
         # GH 10758
-        with pytest.raises(TypeError):
+        msg = "function missing required argument 'day'|Required argument 'day'"
```

Comment:
do you know where these inconsistencies arise?

Reply:
if we only have ```function missing required argument 'day'```
it would only pass the Linux py36_locale_slow_old_np and ```Linux py37_locale``` azure pipeline
The error was raised by pandas/_libs/tslibs/timestamps.pyx:467
```
ts_input = datetime(**datetime_kwargs)
```
Maybe it's a python thing?

Reply:
I know all of our pipeline run on py 3 but when looking around, i noticed that when run
```
open()
```
Python 3 (3.7.3) would returns ```TypeError: open() missing required argument 'file' (pos 1)``` and
Python 2 (2.7.17) would returns ```TypeError: Required argument 'name' (pos 1) not found```

Reply:
ok, this is an indication that the message is not raised by pandas. what's the policy here @MomIsBestFriend?